manifests/jenkins-s2i.yaml

```diff
@@ -12,36 +12,36 @@ parameters:
   - description: Git branch/tag reference for Jenkins S2I
     name: JENKINS_S2I_REF
     value: main
-  - description: Source imagestream
-    name: JENKINS_S2I_SRC_IMAGESTREAM_NAME
-    value: jenkins:scheduled-upgrade-redeploy
-  - description: Namespace of source imagestream
-    name: JENKINS_S2I_SRC_IMAGESTREAM_NAMESPACE
-    value: openshift
+  - description: Pull spec for Jenkins image
+    name: JENKINS_S2I_PULL_SPEC
+    value: registry.redhat.io/ocp-tools-4/jenkins-rhel9:v4.17.0
```

Comment:
we are hardcoding `v4.17.0` in a lot of places. I wonder if the better thing to do would be to define our own image stream (the image stream is the place where we use `v4.17.0`) and then use that imagestream in all places where we want to use that image as input.

i.e. it's what we had before where we were using the `jenkins:scheduled-upgrade-redeploy` imagestream from the `openshift` namespace, except now we just use an imagestream we define here in our own namepsace.